exampleLitActions/src/encryptDecrypt.ts

```diff
@@ -0,0 +1,183 @@
+    console.log("Connecting to Lit Node network...");
+    await litNodeClient.connect();
+    
+    // Get session signatures
+    const sessionSigs = await litNodeClient.getSessionSigs({
+      chain: "ethereum",
+      expiration: new Date(Date.now() + 1000 * 60 * 10).toISOString(), // 10 minutes
+      resourceAbilityRequests: [
+        {
+          resource: new LitActionResource("*"),
+          ability: LIT_ABILITY.LitActionExecution,
+        },
+      ],
+      authNeededCallback: async ({
+        uri,
+        expiration,
+        resourceAbilityRequests,
+      }) => {
+        const toSign = await createSiweMessage({
+          uri,
+          expiration,
+          resources: resourceAbilityRequests,
+          walletAddress: await ethersWallet.getAddress(),
+          nonce: await litNodeClient.getLatestBlockhash(),
+          litNodeClient,
+        });
+        return await generateAuthSig({
+          signer: ethersWallet,
+          toSign,
+        });
+      },
+    });
+
+    // Define access control conditions
+    const chain = 'ethereum' as const; // Specify the type to be a literal, not just a string
+    const accessControlConditions: LitAccessControlCondition[] = [
+      {
+        contractAddress: '',
+        standardContractType: '',
+        chain, 
+        method: 'eth_getBalance',
+        parameters: [':userAddress', 'latest'],
+        returnValueTest: {
+          comparator: '>=',
+          value: '0',
+        },
+      },
+    ];
+
+    // Message to encrypt
+    const messageText = 'Hello world';
+    
+    console.log("Encrypting message:", messageText);
+    
+    // Use the real Lit encryption instead of the mock
+    // This properly encrypts the data and prepares it for the Lit network
+    const { ciphertext, dataToEncryptHash } = await encryptString(
+      {
+        accessControlConditions: accessControlConditions as any, // Type assertion to bypass strict checking
+        dataToEncrypt: messageText,
+        // Remove the chain property as it's not expected in EncryptStringRequest
+      },
+      litNodeClient as any, // Type assertion to bypass strict checking
+    );
+    
+    console.log("cipher text:", ciphertext, "hash:", dataToEncryptHash);
+    
+    // Execute the Lit Action with the necessary parameters
+    console.log("Executing Lit Action to decrypt...");
+    const executeResponse = await litNodeClient.executeJs({
+      code: litActionCode,
+      sessionSigs,
+      // all jsParams can be used anywhere in your litActionCode
+      jsParams: {
+        accessControlConditions,
+        ciphertext,
+        dataToEncryptHash,
+      },
+    });
+    
+    console.log("execute response: ", executeResponse);
+    
+    // Get the decrypted message from the response
+    const decryptedMessage = executeResponse.response || null;
+    
+    // Disconnect from the Lit Node network
+    console.log("Disconnecting from Lit Node network...");
+    await litNodeClient.disconnect();
```

Comment:
This should happen within a `finally` clause like in `exampleLitActions/src/conditionalSigning.ts`